mypy/util.py

```diff
@@ -5,8 +5,15 @@
 import subprocess
 import sys
 
-from typing import TypeVar, List, Tuple, Optional, Dict, Sequence, Iterable, Container
-from typing_extensions import Final, Type
+from typing import TypeVar, List, Tuple, Optional, Dict, Sequence, Iterable, Container, IO
+from typing_extensions import Final, Type, Literal
+
+try:
+    import curses
```

Comment:
Instead of using curses which doesn't support Windows well, perhaps we could use [colorama](https://github.com/tartley/colorama) instead? If you don't want to change things in this PR I can open one to redo this in colorama if you want (maybe after the daemon changes).